doc/build-unix.md

```diff
@@ -8,10 +8,9 @@ To Build
 ---------------------
 
 ```bash
-./autogen.sh
-./configure
-make # use "-j N" for N parallel jobs
-make install # optional
+cmake -B build
```

Comment:
Might it be nice to add a comment that these commands are run from the root of the project's source code?

Reply:
I'm not oppose to this suggestion, but this PR is focused on rather switching build systems than overall improving.

However, I'll support such an improvement once it is PR'ed to the main repository.